Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.7-slim-buster
+LABEL maintainer="Aspergillus Developers <aspergillus@mail.computational-biology.org>"
+
+WORKDIR /opt/nlisim
+COPY nlisim /opt/nlisim/nlisim
+COPY setup.py setup.cfg /opt/nlisim/
+COPY .git /opt/nlisim/.git
+RUN apt update \
+    && apt install -y git libgl1 \
+    && pip3 install . \
+    && apt remove -y git
```

Comment:
If we only need `git` temporarily to do the install, we should be doing the install using [a multi-stage Docker build](https://docs.docker.com/develop/develop-images/multistage-build/).

That's not strictly necessary, but will leave less clutter in the built image.